.changeset/flat-snakes-hammer.md

```diff
@@ -0,0 +1,50 @@
+---
+"@astrojs/vercel": minor
+---
+
+Introduces a new config options, `isr`, that allows to deploy your project as an ISR function. [ISR (Incremental Static Regeneration)](https://vercel.com/docs/incremental-static-regeneration) caches your on-demand rendered pages in the same way as prerendered pages after first request.
+
+To enable this feature, set `isr` to true in your `astro.config.mjs`:
```

Comment:
```suggestion
To enable this feature, set `isr` to true in your Vercel adapter configuration in `astro.config.mjs`:
```
Even though this is an *adapter's* minor, and not an Astro one, I still do kind of want to kind of emphasize that this is a specific adapter setting, and unique to the Vercel adapter, to avoid any confusion.